app/src/androidTest/java/org/mozilla/fenix/ui/NavigationToolbarTest.kt

```diff
@@ -138,4 +141,46 @@ class NavigationToolbarTest {
             verifyFindNextInPageResult("1/1")
         }.closeFindInPage { }
     }
+
+    @Test
+    fun verifySecurePageSecuritySubMenuTest() {
+        val defaultWebPage = "https://mozilla-mobile.github.io/testapp/loginForm"
+        val defaultWebPageTitle = "Login_form"
+        val connectionSecurity = "Connection is secure"
```

Comment:
Can we use `id/securityInfo`?

Reply:
Switched to use the string resource `quick_settings_sheet_secure_connection_2`